src/Symfony/Bridge/PhpUnit/Tests/ClockMockTest.php

```diff
@@ -22,14 +22,12 @@
  */
 class ClockMockTest extends TestCase
 {
-    use ForwardCompatTestTrait;
```

Comment:
If fixed that test in this PR because travis uses files in `src/Symfony/Bridge/PhpUnit` to test this component while other components are tested with master's branche.